src/assets/error/Error.tsx

```
@@ -0,0 +1,45 @@
+import WarningSign from "@/assets/illustrations/warning-sign.svg?react";
+import Button from "@/components/button/Button";
+import { useNavigate } from "react-router-dom";
+import Path from "@/types/paths";
+import { useTranslation } from "react-i18next";
+
+interface ErrorProps extends PropsWithChildren {
+  redirectTo?: string;
+  redirectButtonValue?: string;
+}
+
+const Error = ({
```

Biome

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)